NextJs/pages/index.js

```diff
@@ -1,87 +1,168 @@
-import Head from 'next/head'
-import Image from 'next/image'
-import styles from '../styles/Home.module.css'
+import Head from "next/head";
+import Image from "next/image";
 import React, { useEffect, useState } from "react";
 import axios from "axios";
 import Script from "next/script";
+import Modal from "react-modal";
+import Receipt from "../components/receipt";
+import { Dropdown } from "react-bootstrap";
 
-const merchantPublicKey = "pub_sandbox_N3JQzyuR.VXjgMrW90DWWfwbd2eerxDA3mSG7uVdR";
-const mandoPublicKey = 'pub_sandbox_96qsBHrP.IIVG6xPaGkLIneXeekspk9muIAfYaesn'
+const merchantPublicKey =
+  "pub_sandbox_N3JQzyuR.VXjgMrW90DWWfwbd2eerxDA3mSG7uVdR";
+
```

Comment:
@ali-cash define here the base URLs so that if a client wants to clone this repo and use it with production can just change the constant here for example